src/backend/joanie/core/filters/admin.py

```diff
@@ -70,6 +74,43 @@ def filter_by_query(self, queryset, _name, value):
         """
         return queryset.filter(translations__title__icontains=value).distinct()
 
+    def filter_by_state(self, queryset, _name, value):
+        """
+        Filter resource by looking for states that match the one provided
+        in the "state" parameter
+        """
+        now = django_timezone.now()
+        if value == models.CourseState.ONGOING_OPEN:
+            queryset = queryset.filter(
+                start__lt=now, end__gt=now, enrollment_end__gt=now
```

Comment:
open also requires enrollment_start__lte=now

Reply:
The code for this in compute_state is
```
if start < now:
    if end > now:
        if enrollment_end > now:
            # ongoing open
            return CourseState(CourseState.ONGOING_OPEN, enrollment_end)
```
Is it assumed that if the course is active and the enrollment_end is greater than now, enrollments are open ?
I'll add the check anyway,  seems logical